packages/components/src/components/hds/app-side-nav/index.ts

```diff
@@ -136,6 +138,30 @@ export default class HdsAppSideNav extends Component<HdsAppSideNavSignature> {
     });
   }
 
+  lockBodyScroll(): void {
+    if (this.body) {
+      // Store the initial `overflow` value of `<body>` so we can reset to it
+      this.bodyInitialOverflowValue =
+        this.body.style.getPropertyValue('overflow');
```

Comment:
thought: the initial value could be stored only once, in the `didInsert`, right?

Reply:
Pushed up refactored code